var/spack/repos/builtin/packages/libev/package.py

```diff
@@ -0,0 +1,41 @@
+##############################################################################
+# Copyright (c) 2013-2017, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Libev(AutotoolsPackage):
+    """A full-featured and high-performance event loop that is loosely modelled
+    after libevent, but without its limitations and bugs."""
+
+    homepage = "http://software.schmorp.de/pkg/libev.html"
+    url      = "https://github.com/enki/libev"
+
+    version('develop', git='https://github.com/enki/libev',
+            commit='93823e6ca699df195a6c7b8bfa6006ec40ee0003')
```

Comment:
One last request. Can you add a stable release as well? We like to see at least one stable release in every package. If you add:
```python
url      = "http://dist.schmorp.de/libev/libev-4.24.tar.gz"
list_url = "http://dist.schmorp.de/libev/Attic/"

version('4.24', '94459a5a22db041dec6f98424d6efe54')
```
the `list_url` will allow `spack versions` and `spack checksum` to pick up every version.

Reply:
Fixed.